students/FredB/session03/mailroom.py

```diff
@@ -0,0 +1,101 @@
+#Mailroom Part1
```

Comment:
Nice job Fred, this looks good! Good usage of the os module to clear the screen, keeps it very neat!